### PR TITLE
Status and history commands (In-progress)

### PR DESCRIPTION
**Pull request by khiangleon**

### CODE REVIEW

File: src/main/java/seedu/revision/ui/MainWindow.java [hunk 1]
```diff
@@ -71,6 +71,32 @@ public void handleStart(Mode mode) throws CommandException {
...
     }
 
+    /**
+     * Shows history of results as a line graph. Loads the window components for line graph.
+     * @throws CommandException
+     */
+    @FXML
+    public void handleHistory() throws CommandException {
+        if (this.logic.getStatisticsList().size() > 0) {
+
```
Comment (wilfredbtan) [inline src/main/java/seedu/revision/ui/MainWindow.java:81]: What's supposed to be in this if statement ah?

File: src/main/java/seedu/revision/ui/MainWindow.java [hunk 2]
```diff
@@ -71,6 +71,32 @@ public void handleStart(Mode mode) throws CommandException {
...
+    }
+
+    /**
+     * Shows results of latest quiz attempted as a pie chart. Loads the window components for pie chart.
+     * @throws CommandException
+     */
+    @FXML
+    public void handleStats() throws CommandException {
+        if (this.logic.getStatisticsList().size() > 0) {
+
```
Comment (wilfredbtan) [inline src/main/java/seedu/revision/ui/MainWindow.java:94]: This one also

File: src/main/java/seedu/revision/ui/StartQuizWindow.java [hunk 3]
```diff
@@ -58,7 +59,24 @@
...
+    //score of level 2 difficulty questions
+    private int score2 = 0;
+    //score of level 3 difficulty questions
+    private int score3 = 0;
+    //total number of questions answered in level 1
+    private int total1 = 0;
+    //total number of questions answered in level 2
+    private int total2 = 0;
+    //total number of questions answered in level 3
+    private int total3 = 0;
```
Comment (wilfredbtan) [inline src/main/java/seedu/revision/ui/StartQuizWindow.java:77]: Possible to store this in a dictionary format or an array? Doesn't look very nice code quality wise ah. Can consider that data structure to pass into your Statistics Class as well.

File: src/main/java/seedu/revision/ui/StartQuizWindow.java [hunk 4]
```diff
@@ -255,7 +298,15 @@ public Void call() {
...
         fillInnerParts();
+        score1 = 0;
+        score2 = 0;
+        score3 = 0;
+        total1 = 0;
+        total2 = 0;
+        total3 = 0;
+        totalScore = 0;
         score = 0;
+        accumulatedSize = 0;
```
Comment (wilfredbtan) [inline src/main/java/seedu/revision/ui/StartQuizWindow.java:309]: If using a dictionary, can just clear the dictionary or create a new blank one. might make more sense than this.

File: src/main/java/seedu/revision/ui/MainWindow.java (hunk 1 above)
Comment (khiangleon) [reply src/main/java/seedu/revision/ui/MainWindow.java:81]: supposed to show line graph, still task ongoing

File: src/main/java/seedu/revision/ui/MainWindow.java (hunk 2 above)
Comment (khiangleon) [reply src/main/java/seedu/revision/ui/MainWindow.java:94]: supposed to show pie chart, task still ongoing

File: src/main/java/seedu/revision/ui/StartQuizWindow.java (hunk 3 above)
Comment (khiangleon) [reply src/main/java/seedu/revision/ui/StartQuizWindow.java:77]: ok will look into it

File: src/main/java/seedu/revision/ui/StartQuizWindow.java (hunk 4 above)
Comment (khiangleon) [reply src/main/java/seedu/revision/ui/StartQuizWindow.java:309]: ok I will look into it just like the previous comment

**Pull request merged**